lib/classes/CLI.test.js

```diff
@@ -524,6 +524,18 @@ describe('CLI', () => {
 
       expect(inputToBeProcessed).to.deep.equal(expectedObject);
     });
+
+    it('should be able to parse --ignore-metadataservice-validation', () => {
+      cli = new CLI(serverless, ['deploy', '--ignore-metadataservice-validation']);
+      const inputToBeProcessed = cli.processInput();
+
+      const expectedObject = {
+        commands: ['deploy'],
+        options: { 'ignore-metadataservice-validation': true },
+      };
+
+      expect(inputToBeProcessed).to.deep.equal(expectedObject);
```

Comment:
I wouldn't add this test as It seems redundant (we do not whitelist options one by one, all that are passed to CLI are exposed)

Reply:
I removed the test.